### PR TITLE
Add release notes for 2.10.0

### PR DESCRIPTION
**Pull request by mahipv**
# Description

Update the release notes with the 2.10.0 changelog.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:30]:
```suggestion
**Old format of continuous aggregates deprecation announcement**
```

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:32]:
```suggestion
All instances with continuous aggregates using the old format should [migrate to the new format](https://docs.timescale.com/api/latest/continuous-aggregates/cagg_migrate/) by July 2023,
```

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:34]:
```suggestion
Before that time, we will announce the specific version of TimescaleDB in which support for this feature will be removed.
```

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:56]:
```suggestion
* Hierarchical continuous aggregates (continuous aggregate on top of another continuous aggregate)
```

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:45]:
```suggestion
**Bug fixes**
```

File: timescaledb/overview/release-notes/index.md
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:28]:
```suggestion
We will continue supporting PostgreSQL 12 until July 2023. Before that time, we will announce the specific version of TimescaleDB in which PostgreSQL 12 support will be removed.
```

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: Changed

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: Changed

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: Changed

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md:28]: Changed

**Comment by jnidzwetzki** [comment]
@Loquacity Thanks for the review. I addressed your comments in the most recent version of the PR.

File: timescaledb/overview/release-notes/index.md
Comment (iroussos) [reply timescaledb/overview/release-notes/index.md:28]: @jnidzwetzki I don't see this change on the latest version of the release notes

File: timescaledb/overview/release-notes/index.md
Comment (iroussos) [inline timescaledb/overview/release-notes/index.md:23]: @Loquacity for your review as well

```suggestion
* Joins in continuous aggregates that are defined over hypertables. Support for joins in [hierarchical continuous aggregates](https://docs.timescale.com/timescaledb/latest/how-to-guides/continuous-aggregates/hierarchical-continuous-aggregates/) will be introduced on a follow-up release.
```

**Review by iroussos — CHANGES_REQUESTED** [review]
Thank you @mahipv and @jnidzwetzki, I have added a suggestion for Lana to review, but other than that everything looks good to me.

Once we have all the changes approved, I'll take over the timing for merging the PR so that we can synchronize with the availability of the release on Timescale Cloud and MST. /cc @jfjoly

**Comment by iroussos** [comment]
@nenadmalic as we are going to use this version of the release notes as our Single Source of Truth for our communications, do you mind also reviewing? Please add anyone else that you'd want to review as well. 

If you want to make the source code more easily readable, you can turn off the annotations by the linter:

image

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: @iroussos Thank you for pointing this out. I have forgotten to push this change.

File: timescaledb/overview/release-notes/index.md
Comment (jnidzwetzki) [reply timescaledb/overview/release-notes/index.md]: @iroussos Added

Comment (Loquacity) [reply timescaledb/overview/release-notes/index.md]: LGTM 🚀

**Review by iroussos — APPROVED** [review]

**Comment by iroussos** [comment]
@Loquacity we are ready to merge this - can you help me and merge the PR? the linter is once more blocking the merge :-( 

General question - Should we turn the prose linter off for the releases directory or replace it in general? It generates too many false positives

**Comment by Loquacity** [comment]
> @Loquacity we are ready to merge this - can you help me and merge the PR? the linter is once more blocking the merge :-(
> 

Happy to merge for you now 👍 

> General question - Should we turn the prose linter off for the releases directory or replace it in general? It generates too many false positives

Personally, I would rather get the release notes up to our documentation standards.

**Review by Loquacity — APPROVED** [review]

**Comment by iroussos** [comment]
Thank you @Loquacity!

> Personally, I would rather get the release notes up to our documentation standards.

I agree with that! To add to what you said, note that we are switching to using the release notes as the Single Source of Truth; we'll stop sending the release newsletter and ask everyone interested to subscribe to notifications on this file :-) 

So everything new should be up to the standards of the rest of our documentation.

But the problem here is the long list of all the past release notes that coincide on the same file. If my understanding is correct, we hit the warnings limit way before we reach the latest release notes. 

1. One option would be to do a pass and fix everything, so that going forward we start with as few warnings as possible from the past releases and we'll be able to merge. But that would require someone to go through the warning one by one and fix the issues, which seems like a lot of work to me.
2. The other option would be to split the files to two. Let's say pre and post 2.9 and then link to the old file from the new one. Something like 'Past releases'. That way we can shelf the past release notes and start over with a clean file.

What do you think?

**Comment by Loquacity** [comment]
> What do you think?

We're about to reorganise the docs site, so let's go with your option number 2. I'll set up the files accordingly 👍

**Comment by iroussos** [comment]
> We're about to reorganise the docs site, so let's go with your option number 2. I'll set up the files accordingly 👍 I can also create a template for the notes, too, which might help. It's the same errors over and over again that I fix in these 😉

Sounds great Lana, thank you 🚀

**Comment by Loquacity** [comment]
> Sounds great Lana, thank you 🚀

https://github.com/timescale/docs/pull/2097

**Pull request merged**
